.github/workflows/develop-pull-request-end-to-end-tests.yml

```diff
@@ -101,7 +101,7 @@ jobs:
         run: yarn build-backend
 
       - name: run webpack
-        run: npm run build-local-login
+        run: npm run build-local-login-dev
```

Comment:
Why dev? I'd rather have the tests run on a build that is as it is actually being used on the server. The extra minute building is nothing on the entire time of the tests after all :)